### ISSUE TITLE
step_startup_force Does not work from a cold-boot.

### ISSUE THREAD

**Issue by Nathilion** [ptpd/ptpd#37]
Hello,

For one of our experimental sorting machines I make use of ptpd to keep a couple of PCs synchronized without the aid of a (grand)master clock and so at boot there's always some time difference that needs to be smoothed out. As the PCs are meant to be modular and have no special or distinguishing manual configurations the ptp daemons have been configured to be master-slaves and I wrote a startup script so that it would boot together with the OS.

After the fight on who's going to be the master has been won by one of the PCs I noticed that bigger time differences cause the slaves to step straight away.  Which is perfectly acceptable as I want my time to be synchronized ASAP. Smaller differences can sometimes take up to half an hour to smooth out because the clock is slewed instead of stepped, which is unacceptable.

After some more careful examination of the documentation I noticed the _step_startup_force_ option and thought that this is exactly what I needed. And it is. But only when I manually restart the daemons. When I start the sorting machine by switching the power back on the daemons do not perform this sync once to get into the slave state. When I next restart the daemon or the PC the clock is stepped properly again.

So, long story short: step_startup_force does not work from a cold boot.

This concerns ptpd2 version 2.3.2. Running on Ubuntu 14.04.5
